cpp/include/functions.hpp

```diff
@@ -77,4 +78,23 @@ std::unique_ptr<rmm::device_buffer> renumber_vertices(
   ET *map_size,
   rmm::mr::device_memory_resource *mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief    Broadcast using handle communicator
+ *
+ * Use handle's communicator to operate broadcasting.
+ *
+ * @throws                    cugraph::logic_error when an error occurs.
+ *
+ * @tparam value_t            Type of the data to broadcast
+ *
+ * @param[out] value          Point to the data
+ * @param[in]  count          Number of elements to broadcast
+ *
+ */
+
+template <typename value_t>
+void comms_bcast(const raft::handle_t &handle, value_t *value, size_t count)
+{
+  handle.get_comms().bcast(value, count, 0, handle.get_stream());
+}
```

Comment:
I think we'd better avoid adding an additional layer of indirection without clear benefit.

```comms_bcast(...)``` is not really that more readable than ```handle.get_comms().bcast(...)``` And I can immediately figure out what handle.get_comms().bcast() is but if I see ```comms_bcast``` I should jump to this code to see what this actually does.

Reply:
Just add a FIXME comment saying this is a temporary solution and this function will be removed. Ideally, this should be performed using Dask scatter (if we can close the performance gap). If not, we need to provide more consistent solution to call RAFT communication collectives from python/cython.

Reply:
Added a FIXME in 40e7349